gff/models/two_backbones.py

```diff
@@ -39,8 +38,8 @@ def get_empty_norms(era5_bands: int, era5l_bands: int, glofas_bands: int, hydroa
 
 
 options_metnet = {
-    "dim": 64,
-    "attn_depth": 12,
+    "dim": 32,
+    "attn_depth": 8,
```

Comment:
Wait, why are you changing the metnet configuration?

Reply:
Ah yes, it was too large for my gpu at the time, i can remove this change